docs/releasenotes/release-1.118.md

```diff
@@ -1,26 +1,26 @@
-# v1.118.0
+# v1.118.1
 
-** This version is not yet released; this document is gathering release notes for the future release **
+* This release introduces a new mechanism to reconcile KCC resource. Currently it only applies to `LoggingLogMetric`.
 
-* ...
+* Special shout-outs to @199201shubhamsahu,@acpana,@anhdle,-sso@barney,-s@cheftako,@gemmahou,@GoogleCloudPlatform,@jingyih,@justinsb,@katrielt,@vmiglani,@xiaoweim and @yuwenma for their
```

Comment:
nit - misplaced comma, should be anhdle-sso, barney

Also, I don't think we need to thank GoogleCloudPlatform, unless it's in anticipation of the robots uprising!

Reply:
🤣 🤣 🤣 🤣  my script was less smart. Thanks!